### PR TITLE
Add error returns if command invalid in RunCommand #124

### LINKED ISSUE #124: Potential bug in InTotoRun
**Description of issue or feature request**:

https://github.com/in-toto/in-toto-golang/blob/02b98c8d4e2220c5fa5b992f39cd167986cede78/in_toto/runlib.go#L293 

We are not handling an empty `cmdArgs` either as `nil` or empty string slice. We should check if our code still works with both and that we are not running into a panic.

Furthermore, we should add tests to cover these edge cases.

### PR DESCRIPTION
**Pull request by u5surf**
*Please fill in the fields below to submit a pull request.  The more
information that is provided, the better.*

**Fixes issue #**:

Fixes #124 

**Description of pull request**:

RunCommand has not validation to delegate to exec.Command, so I added simple validation that has checked  cmdArgs length and nil.

**Please verify and check that the pull request fulfills the following
requirements**:

- [x] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature
I consider that it is a trivial bug fix, so I'm not willing to write CHANGE.LOG
If you feel like to write anything in docs, it is welcome for me about this :-)

### CODE REVIEW

**Comment by shibumi** [comment]
Hi @u5surf  sorry, I just realized we were working on the same PR at the same time :D

File: in_toto/runlib.go [hunk 1]
```diff
@@ -240,6 +240,10 @@ command execution.
 */
 func RunCommand(cmdArgs []string, runDir string) (map[string]interface{}, error) {
 
+	if cmdArgs == nil || len(cmdArgs) == 0 {
```
Comment (shibumi) [inline in_toto/runlib.go:243]: It is actually not that easy.. this would lead to an error if cmdArgs is empty.
The in-toto python implementation allows empty cmdArgs, hence we need a fix in both functions. Like here:
https://github.com/in-toto/in-toto-golang/pull/126/files

**Comment by shibumi** [comment]
@u5surf  do you want to adopt my PR in https://github.com/in-toto/in-toto-golang/pull/126/files? 
My PR is missing more tests for the InTotoRun() function (testing link generation for empty cmdArgs). Do you want to work further on this or shall I? :)

**Comment by u5surf** [comment]
@shibumi 
> My PR is missing more tests for the InTotoRun() function (testing link generation for empty cmdArgs). Do you want to work > further on this or shall I? :)
Close this PR.
I would like to add more tests for the InTotoRun() function base on your PR
